### PR TITLE
Add Facebook to users

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: website/data/users.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- caption: Facebook
+  image: "/images/users/facebook-200x100.png"
+  infoLink: https://facebook.com
+  pinned: true
```
Comment (j-f1) [inline website/data/users.yml]: I feel like we should randomize these on each deploy so that everyone gets their time in the spotlight.

Comment (lipis) [reply website/data/users.yml]: Or maybe randomize them on page load..

Comment (lipis) [reply website/data/users.yml]: #3868

**Comment by lipis** [comment]
Well.. :) I would say we should make some "space" for Facebook.. as it's probably the biggest "customer" of Prettier..

**Comment by lipis** [comment]
@azz I think we should add Facebook now and fix the conflicts in the #3718

**Comment by azz** [comment]
It's not that it will cause conflicts, we'd need to get @orta to create an asset for it.

**Comment by orta** [comment]
That's trivial, I can do it whenever

**Comment by lipis** [comment]
Thanks @orta!

**Comment by lipis** [comment]
Are we doing this? @orta did you have time to fix the facebook.svg?

**Comment by orta** [comment]
I was waiting on the website update

**Comment by orta** [comment]
Here we are: https://ortastuff.s3.amazonaws.com/prettier/prettier-facebook-logo.svg

**Comment by lipis** [comment]
Thanks @orta I'll integrate it in a bit..

**Comment by lipis** [comment]
Wait for it to appear here for testing: https://deploy-preview-3867--prettier.netlify.com

**Comment by lipis** [comment]
@azz can you check it?

**Comment by azz** [comment]
Curious why all the other svgs have been changed?

**Comment by lipis** [comment]
For some reason the `svgo` changed them again.. will check them in a separate PR and add the ability to prettify single svg files

**Comment by azz** [comment]
Btw, please use your fork for PR branches.

**Comment by j-f1** [comment]
@azz Any particular reasons for that, or is it just convention?

**Comment by azz** [comment]
Mostly convention. Nice to have branch ownership - e.g. you only force-push or delete branches on your own fork/repo.

**Pull request merged**